src/commands/provar/metadatacache.ts

```diff
@@ -81,6 +81,22 @@ export default class MetadataCache extends SfdxCommand {
         const connections: string = this.flags.connections;
         const connectionoverrides: string = this.flags.connectionoverrides;
 
+        if (
+            !metadataLevel || !cachePath
+        ) {
+            this.ux.error(
+                "ERROR running provar:metadatacache :  Please specify a cachepath and metadatalevel"
+            );
+            return {};
+        }
+
+        (!["Reload", "Refresh", "Reuse"].includes(metadataLevel)) {
```

Comment:
Should this be an if statement?

`if (!["Reload", "Refresh", "Reuse"].includes(metadataLevel)) {`